### PR TITLE
y-axis fixed & ensure operates have parents

### PR DESCRIPTION
**Pull request by NazerkeBS**

### CODE REVIEW

File: html/dashboard.html [hunk 1]
```diff
@@ -3,124 +3,115 @@
...
+          {
+            "type": "stratify",
+            "key": "addr",
+            "parentKey": "parent"
+          },
+          {
+            "type": "partition",
+            "field": "elapsed_ns",
+            "sort": {"field": "value"},
+            "size": [{"signal": " 2 * PI"}, {"signal": "70*PI"}],
```
Comment (utaal) [inline html/dashboard.html:68]: Why this change? I.e. why is the radius of the circle a number multiplied by PI?
Should this be an absolute number in pixels?

Comment (NazerkeBS) [reply html/dashboard.html:68]: it's a typo while copying & pasting. Intended to replace any fixed number. I.e:  "size": [{"signal": "2 * PI"}, 700], or  "size": [{"signal": "2 * PI"}, {"signal": "width / 2"}],

**Comment by utaal** [comment]
Superseded by #12.
